pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go

```diff
@@ -182,42 +192,56 @@ func (r *ReconcileWindowsMachineConfig) Reconcile(request reconcile.Request) (re
 	//		jira story: https://issues.redhat.com/browse/WINC-280
 	windowsNodes, err := r.k8sclientset.CoreV1().Nodes().List(metav1.ListOptions{LabelSelector: nodeconfig.WindowsOSLabel})
 	if err != nil {
+		// This is most likely a permission error, do not requeue
+		log.Error(err, "unable to get count of Windows nodes")
 		return reconcile.Result{}, nil
 	}
 
 	// Get the current count of required number of Windows VMs
 	currentCountOfWindowsVMs := len(windowsNodes.Items)
-	if instance.Spec.Replicas != currentCountOfWindowsVMs {
-		// TODO: We're swallowing the error which is a bad pattern, let's clean this up
-		//		Jira story: https://issues.redhat.com/browse/WINC-266
-		if !r.reconcileWindowsNodes(instance.Spec.Replicas, currentCountOfWindowsVMs) {
-			return reconcile.Result{}, nil
-		}
-	}
+	r.statusMgr.status.JoinedVMCount = currentCountOfWindowsVMs
```

Comment:
Isn't joinedVMCount part of status? Shouldn't we update it as part of statesUpdate call?